loader_hub/jira/base.py

```diff
@@ -1,75 +1,104 @@
-from typing import List
+from typing import List, Optional, Dict, TypedDict
 
 from llama_index.readers.base import BaseReader
 from llama_index.readers.schema.base import Document
 
+
+class basic_auth(TypedDict):
```

Comment:
can we type name with camel case and not snake case? 

class BasicAuth(TypedDict)

class OAuth(TypedDict):

Reply:
I will do the changes.